apps/passport-client/pages/index.tsx

```diff
@@ -439,6 +443,34 @@ async function loadInitialState(): Promise<AppState> {
     serverStorageRevision: persistentSyncStatus.serverStorageRevision,
     serverStorageHash: persistentSyncStatus.serverStorageHash
   };
+
+  if (!validateAndLogStateErrors(state.self, state.identity, state.pcds)) {
+    state.userInvalid = true;
+  } else {
+    const extraValidationErrors = [];
+
+    // this case covers a logged in user. the only way the app can get a 'self'
+    // is by requesting one from the server, to do which one has to be logged in.
+    if (state.self) {
+      if (!state.encryptionKey) {
```

Comment:
I saw a case in loadAfterLogin dealing with a "Zupass legacy login" with no encryption key.  Is that a valid case we need to handle?  If so, maybe this line isn't appropriate for those users.

Reply:
OK. I've removed the encryption key check.